kibbeh/src/modules/user/EditProfileModal.tsx

```diff
@@ -131,7 +131,7 @@ export const EditProfileModal: React.FC<EditProfileModalProps> = ({
                 name="username"
               />
               <InputField
-                className={`mb-4`}
+                className={`mb-4 resizable-textarea`}
```

Comment:
i don't think re-sizing is what we need since it's mostly associated with input. make it scroll instead?

Reply:
also ask @goldyydev about this since i think he was the one that add the current limit

Reply:
It's already scrollable. But you can never look at the whole bio at once.